src/api/resources/sampleunitmethods/benthicphotoquadrattransectmethod.py

```diff
@@ -81,6 +81,26 @@ class BenthicPhotoQuadratTransectMethodView(
     serializer_class = BenthicPhotoQuadratTransectMethodSerializer
     http_method_names = ["get", "put", "head", "delete"]
 
+    def edit_sample_unit(self, request, pk):
+        bpqt_record = self.get_queryset().get(pk=pk)
```

Comment:
Wondering if we should protect against a NotFound exception here or further upstream